packages/zoe/src/types.js

```diff
@@ -335,12 +336,20 @@
  */
 
 /**
+ * @typedef {Object} StandardTerms
+ * @property {IssuerKeywordRecord} terms.issuers - record with
```

Comment:
I don't think that's how the type syntax works. I think you need to omit `terms.` and just use the field name. Also below.

Reply:
Ah, this is vestigal